tests/tests.cpp

```diff
@@ -56,6 +61,27 @@ size_t checksize(size_t& n, size_t added)
     return n;
 }
 
+TEST(MegaApi, getMimeType)
+{
+
+    vector<unique_ptr<thread>> threads;
+
+    atomic<int> successCount = 0;
+
+    for (int i = 0; i < 20; ++i)
+    {
+        threads.emplace_back(unique_ptr<thread>(new thread([&successCount]() {
+            if (::mega::MegaApi::getMimeType("nosuch") == nullptr) ++successCount;
+            if (::mega::MegaApi::getMimeType("323") == string("text/h323")) ++successCount;
+            if (::mega::MegaApi::getMimeType("zip") == string("application/x-zip-compressed")) ++successCount;
+        })));
+    }
+
+    for (auto& a : threads) a->join();
+
+    ASSERT_EQ(successCount, 60);
```

Comment:
Normally, the expected value goes first

Reply:
I think that is done for == comparisons when there's a risk of the programmer accidentally using = ?   I find naming the main tested thing first is closer to natural language.